src/strikethrough.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license Copyright (c) 2017, CKSource - Rémy Hubscher. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module basic-styles/strikethrough
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import StrikethroughEngine from './strikethroughengine';
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+import strikethroughIcon from '../theme/icons/strikethrough.svg';
+
+/**
+ * The strikethrough feature. It introduces the Strikethrough button and the <kbd>Ctrl+Shift+X</kbd> keystroke.
+ *
+ * It uses the {@link module:basic-styles/strikethroughengine~StrikethroughEngine strikethrough engine feature}.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class Strikethrough extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get requires() {
+		return [ StrikethroughEngine ];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'Strikethrough';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		const editor = this.editor;
+		const t = editor.t;
+		const command = editor.commands.get( 'strikethrough' );
+		const keystroke = 'CTRL+SHIFT+X';
```

Comment:
"""
**Firefox: Set direction using the CTRL/CMD+SHIFT+X keyboard shortcut**, which cycles through LTR and RTL. It does not set the value of the element's dir attribute, and is thus invisible to scripts.
"""

From https://www.w3.org/International/questions/qa-html-dir

We somehow got into this issue here: https://github.com/mozilla/notes/issues/409